.changes/unreleased/bug-fixes-635.yaml

```diff
@@ -0,0 +1,6 @@
+component: build
+kind: bug-fixes
+body: Fix release process
```

Comment:
This probably doesn't deserve a changelog, but it's going to be useful to release a .1 release that hopefully succeeds.